solidity/test/StandardPoolConverter.js

```diff
@@ -135,7 +133,7 @@ describe('StandardPoolConverter', () => {
         const factory = await ConverterFactory.new();
         await contractRegistry.registerAddress(registry.CONVERTER_FACTORY, factory.address);
 
-        networkSettings = await NetworkSettings.new(NETWORK_FEE_WALLET, NETWORK_FEE);
+        networkSettings = await NetworkSettings.new(defaultSender, 0);
```

Comment:
Should be `networkFeeWallet`

Reply:
No it shouldn't. There is no purpose to test the network-fee here.
This is just in order to make the rest of the network operable, as we need a viable `NetworkSettings` contract, and the constructor of that contract takes a valid wallet as input).
Where the network-fee tests are relevant, we use an actual network-fee wallet.